pages/linux/sam.md

```diff
@@ -0,0 +1,27 @@
+# sam
+
+> AWS Serverless Application Model (SAM) CLI.
+
+- Initialize a serverless application:
+
+`sam init`
+
+- Initialize a serverless application with a specific runtime:
+
+`sam init --runtime {{python3.7}}`
+
+- Package a SAM application (Alias of *aws cloudformation package*):
```

Comment:
I'm not sure we need to mention an alias here. Perhaps we could write a page for `aws cloudformation` and put

```
> See also `aws cloudformation`.
```

as a 2nd description line?

Reply:
Great suggestion! For not I have removed the references all together as adding pages for the AWS CLI will take some time as it is pretty large. If there is interest in adding those pages I would be happy to do so and update this page to reference it.